src/model/edge.ts

```diff
@@ -2,6 +2,12 @@ import { uuid } from "arg-services";
 import { Metadata } from "./metadata.js";
 import { Node } from "./node.js";
 import { Userdata } from "./userdata.js";
+import * as protobuf from "arg-services/graph/v1/graph_pb";
+import * as date from "../services/date.js";
+import { GeneratePrimeOptionsArrayBuffer } from "crypto";
+import { AddressInfo } from "net";
+import Module from "module";
```

Comment:
Why are these imports needed? I also do not think that these packages are installed at all?